.env.app

```diff
@@ -0,0 +1,6 @@
+PORT=9000
```

Comment:
maybe it's example but everyone use it in development  - we didn't request this change from devs